src/index.test.ts

```diff
@@ -25,6 +25,8 @@ const httpTestClient: TestClient = {
 
 const testClient: TestClient = httpTestClient;
 
+enableFetchMocks();
```

Comment:
I've tested it and can confirm that It works as intended. All tests pass.

Although I would suggest to perhaps do it like this:
```
beforeAll(() => {
  fetchMock.enableMocks();
});

afterAll(() => {
  fetchMock.disableMocks();
});
```

afterAll() is not necessary so we can omit. It shouldn't interfere with other tests but I think it is cleaner if we keep it? WDYT? Also enableMocks could be in beforeAll() to make it a little more clear when it's being set.